moshi_mlx/pyproject.toml

```diff
@@ -33,3 +33,6 @@ dev = [
     "flake8",
     "pre-commit",
 ]
+
+[project.scripts]
+moshi-server = "moshi_mlx.server:main"
```

Comment:
The name of the endpoint is not server in moshi_mlx !
also the two should have different names in case both are installed.

Reply:
Sorry about that, give me a few minutes to clean it up!